### PR TITLE
Setup goimports for networking repos

### PR DESCRIPTION
**Pull request by ZhiminXiang**

### CODE REVIEW

File: static/golang/net-certmanager.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<html><head>
+    <meta name="go-import" content="knative.dev/net-certmanager git https://github.com/knative/net-certmanager">
+    <meta name="go-source" content="knative.dev/net-certmanager     https://github.com/knative/net-certmanager https://github.com/knative/net-/tree/master{/dir} https://github.com/knative/net-certmanager/blob/master{/dir}/{file}#L{line}">
+</head></html>
```
Comment (mattmoor) [inline static/golang/net-certmanager.html]: Add trailing newline:
```suggestion
</head></html>

```

File: static/_redirects [hunk 2]
```diff
@@ -6,6 +6,7 @@
 /eventing-operator/* go-get=1 /golang/eventing-operator.html 200
 /observability/* go-get=1 /golang/observability.html 200
 /networking/* go-get=1 /golang/networking.html 200
+/net-contour/* go-get=1 /golang/net-certmanager.html 200
```
Comment (mattmoor) [inline static/_redirects]:
```suggestion
/net-certmanager/* go-get=1 /golang/net-certmanager.html 200
```

Comment (ZhiminXiang) [reply static/_redirects]: Done

**Pull request merged**